lgc/patch/PatchBufferOp.cpp

```diff
@@ -733,7 +736,12 @@ void PatchBufferOp::visitPHINode(PHINode &phiNode) {
       const int blockIndex = phiNode.getBasicBlockIndex(block);
       assert(blockIndex >= 0);
 
-      Value *const incomingBufferDesc = m_replacementMap[incomings[blockIndex]].first;
+      Value *incomingBufferDesc = m_replacementMap[incomings[blockIndex]].first;
+
+      if (!incomingBufferDesc) {
+        incomingBufferDesc = UndefValue::get(newPhiNode->getType());
+        m_incompletePhi[std::make_pair(newPhiNode, block)] = incomings[blockIndex];
+      }
```

Comment:
Could you also change the 'offset' part in the following to use this way? adding an extra visit() may not be robust enough. I mean the code around line 770.

Reply:
Sure. Done in latest commit.